core/src/main/java/org/jboss/set/mjolnir/archive/mail/report/InvalidUserId.java

```diff
@@ -0,0 +1,51 @@
+package org.jboss.set.mjolnir.archive.mail.report;
+
+import j2html.tags.DomContent;
+import org.jboss.set.mjolnir.archive.domain.RegisteredUser;
+import org.jboss.set.mjolnir.archive.ldap.LdapScanningBean;
+
+import javax.inject.Inject;
+import javax.naming.NamingException;
+
+import java.io.IOException;
+import java.util.Comparator;
+import java.util.List;
+
+import static j2html.TagCreator.*;
+import static j2html.TagCreator.th;
+
+public class InvalidUserId implements ReportTable {
```

Comment:
I would call this bean InvalidResponsiblePersonsTable - meaning would be clearer to me.